cmd/ingress.go

```diff
@@ -26,7 +28,8 @@ func (a *MethodK8s) InitIngressCommand() {
 				return
 			}
 
-			report, err := ingress.EnumerateIngresses(a.K8Config, types)
+			ctx := context.Background()
```

Comment:
you actually should be able to grab `cmd.Context()` (example [here](https://github.com/Method-Security/methodaws/blob/develop/cmd/ec2.go#L22) from methodaws).

Not going to tag each of the ctx calls below but you'll want to replace all of them.